### DISCUSSION TITLE
Multiple Depth values?

### DISCUSSION THREAD

**Discussion by JorgeFBZ** [kinverarity1/lasio#570] [Q&A]
I know that this is not an appropriate use of lasio but, I am trying to use it to store data from drill holes but the measures of azimut and inclination have a different than samples.
(for example: the deviation is measured every 0.2m and the sample values are from 1.5m core.)

Is possible to create multiple DEPT columns from different curves?

Will be very convenient for store all the drill hole info in one file.

Thanks in advance

**Comment by kinverarity1** [comment]
Probably not possible alas! You could have a pretend depth curve, so that you had four curves e.g. DEPT_AZ, AZIMUTH, DEPT_INCL, INCL, but then all LAS-reading software would get very confused by your data.
